test/parallel/test-util-callbackify.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+const common = require('../common');
+
+// This test checks that the semantics of `util.callbackify` are as described in
+// the API docs
+
+const assert = require('assert');
+const { callbackify } = require('util');
+const { join } = require('path');
+const { execFile } = require('child_process');
+const fixtureDir = join(common.fixturesDir, 'uncaught-exceptions');
+const sentinelValues = [
```

Comment:
We had that conversation...
It's TDD speak...
IMHO it's better than `values` but you can 👍 / 👎